source/dem/particle_particle_linear_force.cc

```diff
@@ -108,7 +105,7 @@ ParticleParticleLinearForce<dim>::calculate_particle_particle_contact_force(
   // Looping over local_adjacent_particles values with iterator
   // adjacent_particles_list
   for (auto &&adjacent_particles_list :
-       local_adjacent_particles | boost::adaptors::map_values)
+       container_manager.local_adjacent_particles | boost::adaptors::map_values)
```

Comment:
Here again I would store the local adjacent particles reference into a local variable to prevent always accessing hte object

Reply:
Since it's not in the loop, is it really relevant? (same for the suggestions above this is one)

Reply:
Well I'd just store the local_adjacent_particles reference in a variable at the beginning of the function and be done with it :).

Reply:
As discussed, I will let this modification in the p-p forces files to you in your PR :)